### ISSUE TITLE
Remove "id" from cloned elements

### ISSUE THREAD

**Issue by tsathianathan** [DataTables/FixedColumns#63]
When cloning elements, the "id" should be removed since creates duplicate ids and causes issues with accessibility.

**Comment by DataTables** [comment]
I thought it did this already actually. Or do you have id's in the cells etc? That would be something that isn't currently removed.

**Comment by tsathianathan** [comment]
Yes, I have ids in the cells.
